magento/magento2
# #30009 Resolve regression from CSP updates

### Description (*)


Resolves regression caused from: https://github.com/magento/magento2/commit/4758b4366c3881ce23272e39ea5f5d5a4417a1d7#diff-64b5862c98e5392810d14d8c4d605cb5

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#30009

### Manual testing scenarios (*)

1. Navigate to magento admin
2. Create a widget
3. Add widget to layered category pages
4. Assign to specific categories
5. Observe selector appears

### Questions or comments


### Contribution checklist (*)
